featuretools/primitives/standard/__init__.py

```diff
@@ -1,2 +1,5 @@
 # flake8: noqa
```

Comment:
same here

Reply:
Same issue - the `import *` causes linting problems